src/reynir_correct/errtokenizer.py

```diff
@@ -2636,6 +2663,10 @@ def number_error(
                                 suggest=correct,
                             )
                         )
+            if suppress_suggestions and token.error_code and token.error_code == "Z001" and isinstance(token.val, list) and any(v.ordfl == "lo" for v in token.val):  # type: ignore
```

Comment:
The first "and token.error_code" can be deleted, the later comparison catches that case